azdev/help.py

```diff
@@ -81,7 +81,16 @@
 
 
 helps['verify version'] = """
-    short-summary: Verify package versions.
+    short-summary: Verify package versions against those hosted on PyPI and/or in the azure-cli setup.py file.
+    long-summary: >
+        This is used to ensure the correct module versions are bumped prior to release and that all
+        module versions are present in the azure-cli setup.py with their versions pinned.
+    examples:
+        - name: Verify all versions and update azure-cli's setup.py with each module's verison.
+          text: azdev verify version
+
+        - name: Verify all versions and audit them against azure-cli's setup.py ONLY.
+          text: azdev verify version --no-update
```

Comment:
Nit, I'd still prefer that the default action was to just audit here, and that providing a flag that said "overwrite" or something similar caused changes to disk. Makes more sense to me when the command is "verify". That said, I with Git and the CI scenarios that we have in mind, it's no big deal.

Reply:
I figured to save you some typing, but I agree with you in principle. I'll make the change :)